manager_D3S.py

```diff
@@ -10,12 +10,15 @@
 import signal
 import sys
 from Crypto.Cipher import AES
+from collections import deque
```

Comment:
Made it a plain list (even though I love deques). The way the code is set up, the the maximum length of the wqueue will be 1, so we never have to worry about the list getting too long.